spec/lib/job_notifier/notifier_spec.rb

```diff
@@ -40,13 +41,14 @@ def self.expect_job_creation
   end
 
   def job_by_identifier(identifier)
+    sleep(1)
     JobNotifier::Job.where(identifier: identifier).first
   end
 
   context "defining perform_with_feedback" do
     context "with success feedback" do
       before do
-        class ImageUploadJob < ActiveJob::Base
+        class ImageUploadJob < ApplicationJob
```

Comment:
⚠️ **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Lint/ConstantDefinitionInBlock: Do not define constants this way within a block.